### DISCUSSION TITLE
Support SSL certificate validation when using https pagerduty_proxy

### DISCUSSION THREAD

**Discussion by kexin-zhai** [jertel/elastalert2#1407] [Ideas]
When we tried to trigger a pagerduty alert by hitting `https://<hostname>:<port>` pagerduty_proxy, we ran into SSLCertVerificationError.
```
elastalert Error while running alert pagerduty: Error posting to pagerduty: HTTPSConnectionPool(host='events.pagerduty.com', port=443): Max retries exceeded with url: /v2/enqueue (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:992)')))
```

When we pass a CA cert bundle to  `slack_ca_certs` to trigger slack alerts hitting the same proxy URL, elastalert works as expected.
I was wondering if we may have a plan to add a `pagerduty_ca_certs` option to validate the SSL certificate when using pagerduty proxy.

Thanks!

**Comment by jertel** [comment]
Are you interested in submitting a PR?

**Reply by kexin-zhai** [reply]
Yeah, we will plan on contributing a PR for it.

**Comment by kexin-zhai** [comment]
PR is created and ready for review for this discussion 🙏 : https://github.com/jertel/elastalert2/pull/1418
